src/main/java/org/openmrs/maven/plugins/utility/PropertyManager.java

```diff
@@ -0,0 +1,79 @@
+package org.openmrs.maven.plugins.utility;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.util.Properties;
+
+/**
+ * Class for reading/writing .properties files
+ */
+public class PropertyManager {
+    private Properties properties;
+    private String path;
+
+    public PropertyManager() { properties = new Properties(); }
+
+    /**
+     * Initialize
+     * @param filePath - .properties file path
+     */
+    public PropertyManager(String filePath) {
+        this();
+        path = filePath;
+        File config = new File(path);
+        if (config.exists()) {
+            FileInputStream in = null;
+            try {
+                in = new FileInputStream(path);
+                properties.load(in);
+                in.close();
+            } catch (IOException e) {
```

Comment:
You should add finally block here to call IOUtils.closeQuitely(in) in case an exception is thrown.